packages/gatsby-plugin-google-tagmanager/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
```

Comment:
In this light I have some new ideas about plugin package structure as well. Because there are standards to distinguish between browser and node parts of a package, I figured that if we can merge the gatsby-ssr and gatsby-node api's in a single entry (since they are both node context), we can use the same technique as other packages use for making the distinction.

You would basically have a `main` field pointing to the merged `gatsby-ssr` & `gatsby-node` entry, and a `browser` field pointing to the `gatsby-browser` entry.

Then in the future when webpack supports subfields in `esnext` you can have `esnext.browser` point to the raw gatsby-browser code etc.  So then plugin packages can export transpiled code and raw code for both contexts at the same time like this:

```json
 "main": "dist/gatsby-node-and-ssr.js",
 "browser: "dist/gatsby-browser.js",
 "esnext": {
      "main": "src/gatsby-node-and-ssr.js",
      "browser": "src/gatsby-browser.js"
 }
```